src/datastore/NedbDatastore.ts

```diff
@@ -17,17 +17,32 @@ import { BridgedRoom } from "../BridgedRoom";
 import { SlackGhost } from "../SlackGhost";
```

Comment:
As per what we said in sync. If the code is done and working then don't remove it, but if it's still not right we don't need to support NeDB.

Reply:
All my manual testing was done with Postgres... there could be plain errors in this NeDB implementation.